app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
This is a Code Style issue. You are raising `NotVaccinatedError` without any message. It is a good practice to provide descriptive error messages for exceptions. It will help other developers to understand the problem quickly. You might want to raise it like this: `raise NotVaccinatedError('The visitor needs to be vaccinated to enter the cafe.')`